CDCgov/prime-reportstream
# Jimmy/8841 adjust etor transforms

This PR only involved re-organization of schemas for Flexion senders/receivers, and updates to the flexion organization settings to point to the new schema locations. This didn't involve more significant changes; for details on why, see the comment on the ticket #8841 

Test Steps:
1. Reload the flexion organization settings
2. Send a report from the etor-service-sender to the simulated-lab receiver
3. Query the submission history to verify that the receiver received the report
4. Check the sftp folder to verify that OBR-11 was set to `P`.

## Changes
- Create a Flexion folder for sender transforms and a placeholder sender transform schema in the folder for Trusted Intermediary
- Move the existing Trusted Intermediary receiver transform schema into a new folder
- Update the organization settings for etor to point to the new schema locations

## Checklist

### Testing
- [N/A] Added tests?

## Linked Issues
- Fixes #8841
